OasisDEX/oasis-borrow
# New vault layout in overview tabs

# New vault layout in overview tabs
Combines three stories:
https://app.shortcut.com/oazo-apps/story/4146/implement-new-vault-details-section-in-borrow-vault
https://app.shortcut.com/oazo-apps/story/4147/implement-new-vault-details-section-in-multiply-vault
https://app.shortcut.com/oazo-apps/story/4148/implement-new-vault-details-section-in-earn-vault
  
## Changes 👷‍♀️
Borrow vault, multiply vault and GUNI vault overview tabs are rewritten into new layout according to figma: https://www.figma.com/file/S6pnwbeRY6T8hfNQKxzp7K/Components?node-id=504%3A2130

- Buttons in header sections are pushed outside of those stories as they content is still in progress according to this discussion: https://www.figma.com/file/kAPyiF2JpYre2pgxiYunEr?node-id=7953:54541#183952531
- Modals with details for each card weren't changed in any way according to acceptance criteria from previous story regarding components used here: https://app.shortcut.com/oazo-apps/story/3856/redesign-update-detail-cards

## How to test 🧪
`AutomationBasicBuyAndSell` feature flag is needed to see any changes, in some cases `Automation` flag might also be required.

Changes involves six pages:
- Open borrow vault,
- Manage borrow vault,
- Open multiply vault,
- Manage multiply vault,
- Open GUNI vault,
- Manage GUNI vault.

EDIT: sonar cloud duplication and codecov code smells reports are here because we are using feature flag and some of the code HAD to be duplicated. All of it is going to be removed as soon as we can remove feature flag along with duplicated lines or entire files 😵
